std_msgs/String fen
---
std_msgs/String bestmove